pkg/tests/mappings/filter_directory/test_filter_directory.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Negative tests for the filter_directory script
+
+For a discussion as for why this needs to exist, see """
+
+import pathlib
+import os
+import sys
+import tempfile
+import unittest
+from rules_python.python.runfiles import runfiles
+
+# Get the filter_directory script into the import path.  There might be a
+# better way to do this, but it works.
+
+rf = runfiles.Create()
+filter_directory_script = rf.Rlocation(os.path.join(
```

Comment:
os.path.join does not port to windows with Rlocation. You have to use   '/' as the separator.

Reply:
Bah, forgot this again.  Sorry about that (yay muscle memory).